cmd/lncli/cmd_pay.go

```diff
@@ -59,6 +64,11 @@ var (
 			"used",
 		Value: 1,
 	}
+
+	jsonFlag = cli.BoolFlag{
+		Name:  "json",
+		Usage: "if set, payment updates are printed as json messages",
```

Comment:
nit: could add `-j` as the short hand

Reply:
Hmm, I am not even convinced that we need json output, let alone a short hand. 

I think it would help to decide at some point what we want `lncli` to be. A text-mode user interface or also a tool that we promote for usage in bash scripts.

Reply:
> A text-mode user interface or also a tool that we promote for usage in bash scripts.

I don't think the distinction needs to be that binary. It can have an improved interface for regular tasks while still provided access to the raw data for debugging.

Reply:
Yes, that is true. Maybe it is indeed good for the most fundamental command of all, `sendpayment`, to have a few options.